compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/LightTreePositioningStrategies.kt

```diff
@@ -247,6 +247,30 @@ object LightTreePositioningStrategies {
             return markElement(tree.operationReference(node) ?: node, startOffset, endOffset, tree, node)
         }
     }
+
+    val PARAMETER_DEFAULT_VALUE: LightTreePositioningStrategy = object : LightTreePositioningStrategy() {
+        override fun mark(
+            node: LighterASTNode,
+            startOffset: Int,
+            endOffset: Int,
+            tree: FlyweightCapableTreeStructure<LighterASTNode>
+        ): List<TextRange> {
+            val defaultValueElement = tree.defaultValue(node) ?: node
+            return markElement(defaultValueElement, startOffset, endOffset, tree, node)
+        }
+    }
+
+    val PARAMETER_VARARG_MODIFIER: LightTreePositioningStrategy = object : LightTreePositioningStrategy() {
```

Comment:
I accidentally used this for `USELESS_VARARG_ON_PARAMETER`, and... after https://github.com/JetBrains/kotlin/commit/40bec30393dea6462ba2548da88296a80d99f15a, this one shows the difference. It turned out that old FE reports that diagnostic on the parameter as a whole. Glad we can find this mistake earlier.

There are other diagnostics that will use this, so we can keep this. At least I can confirm it really marks `vararg` modifier. :)